### DISCUSSION TITLE
Is it possible to have different zoom levels per pane/tab?

### DISCUSSION THREAD

**Discussion by tankorsmash** [wezterm/wezterm#6224] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

wezterm 20240921-124912-c65dc634

### Ask your question!

This is in Windows Terminal:
![image](https://github.com/user-attachments/assets/d0397bd2-b277-4a9f-84c1-4ad5a1486436), where I've got different zoom levels per pane. But I can also do it per tab as well:

https://github.com/user-attachments/assets/4f3fe4d8-13d0-403c-9cf3-601f6b841b40

I've got different processes running in my panes. Sometimes I like my editor to be zoomed in, and sometimes I like the cat'ted logs to be way zoomed out. 
I'd like to be able to do this in Wezterm as well; per pane, and especially per tab. For now, the workaround is to simply repeatedly press Ctrl+ a lot between processes, but having them individually set scales would make things easier for me.

Is such a thing possible?a

**Comment by cjasaMHC** [comment]
Would also like to have this functionality, if not already possible :)
